### ISSUE TITLE
kubeadm not working

### ISSUE THREAD

**Issue by kenroyandfamily** [arush-sal/cka-practice-environment#17]
Following the README it was simple to get up and running at http://localhost (on Mac OS 10.15.7)

But I can't get passed the first question :-)

I ran the following commands:

> apt-get update

sudo is missing so I install it
> apt-get install sudo 

These commands come from https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/
> sudo apt-get update && sudo apt-get install -y apt-transport-https curl
> curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
> cat <<EOF | sudo tee /etc/apt/sources.list.d/kubernetes.list
   deb https://apt.kubernetes.io/ kubernetes-xenial main
   EOF
> sudo apt-get update
> sudo apt-get install -y kubelet kubeadm kubectl
> sudo apt-mark hold kubelet kubeadm kubectl

But when I run "kubeadm init" I get the following error:
W1107 15:30:50.563919    1034 kubelet.go:200] cannot automatically set CgroupDriver when starting the Kubelet: cannot execute 'docker info -f {{.CgroupDriver}}':
executable file not found in $PATH

Basically the "docker" command is not found.

I've tried installing docker, but I get errors when trying to start it.
